tests/test_game.py

```diff
@@ -14,9 +14,9 @@ class TestGame(TestCase):
     def setUp(self):
         dict = {}
         self.game = Game(dict, dict)
-        self.sceneone = Scene("TestScene1")
-        self.scenetwo = Scene("TestScene2")
-        self.scenethree = Scene("TestScene3")
+        self.sceneone = MockUpScene("TestScene1")
```

Comment:
should be self.scene_one, self.scene_two, self.scene_three.

or self.scene1, self.scene2, self.scene3. :) Don't worry about changing this now